docs/_docs/tools/control-script.adoc

```diff
@@ -646,4 +646,50 @@ control.bat --change-tag _<new-tag>_
 ----
 --
 
+== Indexes Management
+
+The commands below allow to get a specific information on indexes and to trigger the indexes rebuild process.
+
+To get the list of all indexes that match specified filters, use the command:
+
+[source, shell]
+----
+--cache indexes_list [--node-id nodeId] [--group-name grpRegExp] [--cache-name cacheRegExp] [--index-name idxNameRegExp]
```

Comment:
The command is incomplete. There needs to be a version for Unix (control.sh) and Windows (control.bat). See how other commands are explained above.

Reply:
Fixed.